sky/clouds/gcp.py

```diff
@@ -393,6 +408,10 @@ def make_deploy_resources_variables(
         assert image_id is not None, (image_id, r)
         resources_vars['image_id'] = image_id
 
+        if self._is_machine_image(image_id):
+            resources_vars['machine_image'] = image_id
```

Comment:
Should we add `'machine_image': None` to L347?

Reply:
Yes! Just discovered concurrently lol. Thansk!